docs/book/src/testing/testing-with-rust.md

```diff
@@ -185,25 +188,22 @@ async fn get_contract_instance() -> (TestContract, ContractId) {
             Some(1_000_000_000), /* Amount per coin */
         ),
         None,
+        None,
     )
-    .await;
+    .await
+    .unwrap();
     let wallet = wallets.pop().unwrap();
 
     let id = Contract::load_from(
         "./out/debug/my-fuel-project.bin",
-        LoadConfiguration::default().set_storage_configuration(
-            StorageConfiguration::load_from(
-                "./out/debug/my-fuel-project-storage_slots.json",
-            )
-            .unwrap(),
-        ),
+        LoadConfiguration::default(),
```

Comment:
This seems to be missing an equivalent `with_storage_configuration` call.

Reply:
> Without that it won't have the same behavior.

The code currently in the book doesn't compile. So I'm not quite sure what you mean.

Could I impose on you to push to my branch?

Reply:
It was updated in master.